flink-architecture-tests/violations/d208df3e-0594-4d6c-82bb-94b12f68ec1f

```diff
@@ -0,0 +1,66 @@
+Constructor <org.apache.flink.runtime.scheduler.exceptionhistory.ExceptionHistoryEntry.<init>(java.lang.Throwable, long, java.lang.String, org.apache.flink.runtime.taskmanager.TaskManagerLocation)> calls method <org.apache.flink.runtime.scheduler.exceptionhistory.ExceptionHistoryEntry$ArchivedTaskManagerLocation.fromTaskManagerLocation(org.apache.flink.runtime.taskmanager.TaskManagerLocation)> in (ExceptionHistoryEntry.java:90)
```

Comment:
Just as an FYI, the line numbers contained at the end here are ignored during violation matching, so we don't need to worry that this will constantly break because code moves around (within the same file).